sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Other Changes
 
+- [[#5141]](https://github.com/Azure/azure-sdk-for-cpp/issues/5141) Added error response details to the Identity exceptions thrown when the authority host returns error response.
```

Comment:
nit: As a reader, I would wonder what are "Identity exceptions". Reword to use the actual exception class, i.e. `AuthenticationException`.

Reply:
#5211